Documentation/policy/language.rst

```diff
@@ -34,7 +34,18 @@ Labels Based
 ------------
 
 Label-based L3 policy is used to establish policy between endpoints inside the
-cluster managed by Cilium.
+cluster managed by Cilium. Label-based L3 policies are defined by using an
+`EndpointSelector` inside a rule to choose what kind of traffic that can be
+received (on ingress), or sent (on egress). An empty `EndpointSelector` allows
+all traffic. The examples below demonstrate this in further detail.
+
+When Cilium is running with kubernetes as an orchestrator, the network policies
+are typically scoped by ``Namespace``. If an explicit namespace selector is not
```

Comment:
it's not "typically", it's always. The network policies are always scoped by namespace because that's where they are enforced. For me, the key word here is the "enforced".